ChangeLog

```diff
@@ -23,6 +23,10 @@ Release Date: TBA
 
   Closes #926
 
+* Add `kind` field to `Const` nodes, matching the structure of the built-in `ast` Const.
```

Comment:
Just notice it, the Changelog ist in a rst format so "kind" and "const" need to be escaped twice (take a look at other entries how it's done there). I would also recommend to remove the escaping from "ast"